### DISCUSSION TITLE
hover to open customise is kind of ruins the user experience

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#29457] [Q&A] [answered]
The current user habit is to click Customize when want to adjust, and then move the mouse downwards to setting.

now hover to open will let click as close, then mouse move down user notice panel is closed, then try to click then again.

I want to have two states for hover (open until lost hoverf) and click (open until the next click)

Currently behavior:

https://github.com/user-attachments/assets/18217884-dfcd-473c-83c9-5db5481d9dc2

I throught it should be:

https://github.com/user-attachments/assets/55f258c0-353c-4400-abe1-d7432dbc4d88

```diff
diff --git a/osu.Game/Overlays/Mods/ModCustomisationHeader.cs b/osu.Game/Overlays/Mods/ModCustomisationHeader.cs
index abd48a0dcb..236d4c67b0 100644
--- a/osu.Game/Overlays/Mods/ModCustomisationHeader.cs
+++ b/osu.Game/Overlays/Mods/ModCustomisationHeader.cs
@@ -119,6 +119,7 @@ protected override bool OnClick(ClickEvent e)
                 ExpandedState.Value = ExpandedState.Value switch
                 {
                     ModCustomisationPanelState.Collapsed => ModCustomisationPanelState.Expanded,
+                    ModCustomisationPanelState.ExpandedByHover => ModCustomisationPanelState.Expanded,
                     _ => ModCustomisationPanelState.Collapsed
                 };
             }
```

**Comment by peppy** [comment]
Probably fine to apply this change in behaviour, I also felt it was a touch weird that it closed on first click.

**Comment by cdwcgt** [comment, marked as answer]
https://github.com/ppy/osu/pull/29459
